mlx/backend/metal/kernels/fast_inference_sdpa_params.h

```diff
@@ -0,0 +1,21 @@
+//
+//  MLXFastInferenceSDPAParams.h
+//  mlx
+//
+//  Copyright (C) 2024 Argmax, Inc.
```

Comment:
I don't know if we can have this in the source code. If you are cool to remove it that would be simplest. O/w will need to check internally that it's ok.

Reply:
cool to remove